src/beneuro_data/cli.py

```diff
@@ -678,6 +742,119 @@ def upload_session(
     return True
 
 
+@app.command()
+def up(
+    session_name: Annotated[
+        str, typer.Argument(help="Name of session: M123_2000_02_03_14_15")
+    ],
+    include_behavior: Annotated[
+        bool,
+        typer.Option(
+            "--include-behavior/--ignore-behavior",
+            "-b/-B",
+            help="Download behavioral data (-b) or not (-B)."
+        ),
+    ] = True,
+    include_ephys: Annotated[
+        bool,
+        typer.Option(
+            "--include-ephys/--ignore-ephys",
+            "-e/-E",
+            help="Download ephys data (-e) or not (-E)."
+        ),
+    ] = False,
+    include_videos: Annotated[
+        bool,
+        typer.Option(
+            "--include-videos/--ignore-videos",
+            "-v/-V",
+            help="Download video data (-v) or not (-V)."
+        ),
+    ] = False,
+    include_extra_files: Annotated[
+        bool,
+        typer.Option(
+            "--include-extra-files/--ignore-extra-files",
+            help="Upload extra files that are created by the experimenter or other software.",
+        ),
+    ] = True,
+    rename_videos_first: Annotated[
+        bool,
+        typer.Option(
+            "--rename/--no-rename",
+            help="Rename videos before validating and uploading.",
+        ),
+    ] = True,
+    rename_extra_files_first: Annotated[
+        bool,
+        typer.Option(
+            "--rename-extra/--no-rename-extra",
+            help="Rename extra files (e.g. comment.txt) before validating and uploading.",
+        ),
+    ] = True,
+    processing_level: Annotated[
+        str, typer.Argument(help="Processing level of the session. raw or processed.")
+    ] = "raw",
+):
+    """
+    Upload (raw) experimental data to the remote server.
+
+    Example usage to upload everything of a given session:
+        `bnd up M017_2024_03_12_18_45 -ev`
+    Example to upload the videos and ephys of the last session of a subject:
+        `bnd up M017 -evB`
+    """
+    animal = session_name[:4]
+
+    if processing_level != "raw":
+        raise NotImplementedError("Sorry, only raw data is supported for now.")
+
+    if all([not include_behavior, not include_ephys, not include_videos]):
+        raise ValueError("At least one data type must be included.")
+
+    # if videos are included, rename them first if not specified otherwise
+    if rename_videos_first is None:
+        rename_videos_first = include_videos
+
+    if rename_videos_first and not include_videos:
+        raise ValueError(
+            "Do not rename videos if you're not uploading them. (Meaning --ignore-videos and --rename-videos are not allowed together.)"
+        )
+
+    config = _load_config()
+
+    if len(session_name) > 4:  # session name is given
```

Comment:
Again, I'd rather not assume this and do
```suggestion
    if session_name is not None:  # session name is given
```